arviz/tests/base_tests/test_utils.py

```diff
@@ -64,6 +57,36 @@ def test_var_names_key_error(data):
         _var_names(("theta", "tau", "bad_var_name"), data)
 
 
+@pytest.mark.parametrize(
+    "var_names_expected",
```

Comment:
given that this is `var_names`, `expected` and `filter`, I would name it `args`

Reply:
I opted for `var_args` -- `args` seemed too generic.